src/ProductBundle/Resources/views/ProductAdmin/create_variation.html.twig

```diff
@@ -1,4 +1,4 @@
-o{#
+{#
```

Comment:
You fixed this already in #600 , I suppose this is because you are not fully up to date with master

Reply:
Yes, i think i've messed when rebase